pkgs/crypto/keys/keybase.go

```diff
@@ -321,16 +323,21 @@ func (kb dbKeybase) ExportPrivKey(name string, decryptPassphrase string,
 // ImportPrivKey imports a private key in ASCII armor format.
 // It returns an error if a key with the same name exists or a wrong encryption passphrase is
 // supplied.
-func (kb dbKeybase) ImportPrivKey(name string, astr string, passphrase string) error {
+func (kb dbKeybase) ImportPrivKey(
+	name string,
+	astr string,
+	decryptPassphrase,
+	encryptPassphrase string,
```

Comment:
inconsistent style.

Reply:
Good catch 💯 

I've updated all methods in this file to match the param code style:

[48aade7](https://github.com/gnolang/gno/pull/442/commits/48aade7f944148a68075ce791bebb0e9ed0bd134)